Documentation/ceph-monitoring.md

```diff
@@ -117,6 +117,21 @@ spec:
   monitoring:
     enabled: true
     rulesNamespace: "rook-ceph"
+    # prometheus rule alerts values for overriding default prometheus rules values
+    # optional: specific alerts can be disabled by setting disabled field to true 
+    #alertRuleOverrides:
```

Comment:
I tested out some custom values and I get an error:
```
2022-01-14 21:10:50.039243 E | op-mgr: failed to deploy prometheus rule. prometheus rule could not be deployed: failed to create prometheusRules. PrometheusRule.monitoring.coreos.com "prometheus-ceph-v16-rules" is invalid: spec.groups.rules: Required value
```

My test values were:
```yaml
    alertRuleOverrides:
      CephNodeDown:
        disabled: true
      CephMgrIsAbsent:
        for: 1m
        severityLevel: warning
        severity: error
```

How do you suggest troubleshooting? It would be very verbose, but perhaps we should write the entire prometheus rules yaml to the log with `logger.Errorf()` if the creation fails. But if there is no more detail on the failure it will be very difficult to track down what required value is missing.

Reply:
If I remove my customizations, the prometheus rules CR gets created as expected, so my customization must be invalid in some way.

Reply:
The reason for the failure in that case was:
When disabling CephNodeDown alert, all the alerts rules in the group "ceph-node-alert.rules" were disabled (CephNodeDown is the only alert in this group).
Prometheus rule cannot be deployed when there is a group without any rules.
Now, after disabling all the alerts in specific groups. These groups will be removed from Prometheus Rule Spec, and the prometheus Rule will be successfully deployed.

Regarding your suggestion about logger.Errorf(), I agree, it might help us understand better what is the cause for the error.